testsuite/tests/system/test_system_redis_secret.py

```diff
@@ -0,0 +1,36 @@
+""" Tests for MessageBus variables in system-redis secret """
+
+import pytest
+from packaging.version import Version  # noqa # pylint: disable=unused-import
+from testsuite import TESTED_VERSION  # noqa # pylint: disable=unused-import
+
+pytestmark = [pytest.mark.issue("https://issues.redhat.com/browse/THREESCALE-7822")]
+
+KEYS = [
+    "MESSAGE_BUS_NAMESPACE",
+    "MESSAGE_BUS_SENTINEL_HOSTS",
+    "MESSAGE_BUS_SENTINEL_ROLE",
+    "MESSAGE_BUS_URL",
+]
+
+
+@pytest.fixture(scope='session')
+def system_redis_secret(openshift):
+    """ return system-redis secret """
+    secret = openshift().secrets['system-redis']
+    return secret
+
+
+@pytest.mark.skipif("TESTED_VERSION >= Version('2.12')")
+@pytest.mark.parametrize("key", KEYS)
+def test_message_bus_secrets(system_redis_secret, key):
+    """ test of env variable presence """
+    assert system_redis_secret[key] is not None
```

Comment:
Would using the `in` operator work? If yes, you could condense it into just one test

Reply:
`in` is not working, as only `__getitem__` is defined.